apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Metrics/DashboardMetricsService.ts

```diff
@@ -94,7 +94,7 @@ export class DashboardMetricsService {
 
     try {
       const allMetricsPromises = [
-        fetchGenericSubgraph(
```

Comment:
now we could add type check here, e.g. check `TxnHistoryTable.tsx` subgraph queries